subsys/bluetooth/audio/ascs.c

```diff
@@ -354,6 +356,29 @@ static void ase_exit_state_streaming(struct bt_ascs_ase *ase)
 	} else {
 		LOG_WRN("No callback for stopped set");
 	}
+	if (next_state == BT_BAP_EP_STATE_QOS_CONFIGURED && ase->ep.dir == BT_AUDIO_DIR_SINK) {
```

Comment:
```suggestion
	}
	
	if (next_state == BT_BAP_EP_STATE_QOS_CONFIGURED && ase->ep.dir == BT_AUDIO_DIR_SINK) {
```

Reply:
Please also add a comment why we do this, as this implements functionality that differs from the state machine